internal/configuration.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import "github.com/caarlos0/env/v11"
+
+type Configuration struct {
+	GithubToken string `env:"GITHUB_TOKEN,notEmpty"`
```

Comment:
Add logging or error messages that provide context when the environment variables are not set or invalid. This will aid in debugging configuration issues quickly during development or deployment.